### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by akflds**
Update the README to describe the project and provide installation instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,49 @@
-# Installation
+# Read Me A Story
+
+Read Me A Story is an app that allows family members record, upload and playback bedtime stories for young children in the family when they are unable to read them in person.
```
Comment (321jellyfish) [inline README.md]: missing 'to' - 'to record, upload' etc.

File: README.md [hunk 2]
```diff
@@ -1,30 +1,49 @@
...
 
-Type these in terminal to run flask/pytest:
+Having setup the project you can start the local server:
 
 ```
 flask run
-pytest
 ```
+
+By default the server will start on [http://127.0.0.1:5000](http://127.0.0.1:5000). If you want to run the server in debug mode, set set `FLASK_ENV=development` before starting Flask.
```
Comment (321jellyfish) [inline README.md]: Word 'set' repeated?

File: README.md [hunk 3]
```diff
@@ -1,30 +1,49 @@
-# Installation
+# Read Me A Story
+
+Read Me A Story is an app that allows family members record, upload and playback bedtime stories for young children in the family when they are unable to read them in person.
+
+A parent can sign up to the app, create a family group and invite their partner or relatives to join. Family members can then choose to record and upload stories, or view a list of stories created by the family. Stories can be selected from the list and played back to the child.
+
+Read Me A Story was created by Minimum Viable Panic as their final project in the [Northcoders](https://northcoders.com) coding bootcamp. To find out more about the project check out the [product demonstration](#) or head on over to the [live app](#).
+
```
Comment (321jellyfish) [inline README.md:8]: should we have a link to the front end repo too?

Comment (akflds) [reply README.md:8]: Have reworded this section to include link

**Review by 321jellyfish — APPROVED** [review]
Well dun

**Pull request merged**